### PR TITLE
Blog App - Views

### PR DESCRIPTION
**Pull request by muhammedb8826**
### In this project I have worked on the followings:

- created a view for user controller actions index and show
- created a view for post controller actions index and show
- Used Bootstrap for styling

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,26 @@
...
+        <sub>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %> </sub>
+    </div>  
+    <div class="col col-12 border border-3 border-black">
+        <span>Comments</span>
+        <% post.comments.each do |comment| %>
+        <p><%= comment.text %></p>
+        <% end %>
+    </div>
+    <% end %>
+  </div>
```
Comment (DeliceLydia) [inline app/views/posts/index.html.erb]: - [ ] On the user posts page you should render the user's 3 most recent post and i suggest that you add `See all Post` link to be able to view all posts of the user

Comment (DeliceLydia) [reply app/views/posts/index.html.erb]: - [ ] One post should render 5 most recent comments

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @muhammedb8826, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The Views are associated with the Models and Controllers
- No linters errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (muhammedb8826) [reply app/views/posts/index.html.erb]: Thank you for the feedback, I will do that

**Review by fobadara — APPROVED** [review]
Hi @muhammedb8826, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Proper implementation of reviewer's requirements
✅ Clean code

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muhammedb8826** [comment]
Thank you

**Pull request merged**